suricata.yaml.in

```diff
@@ -1055,7 +1055,8 @@ vlan:
 # of that flow). The value of "established" is the amount of
 # seconds that the engine will wait to free the flow if it spend that amount
 # without receiving new packets or closing the connection. "closed" is the
-# amount of time to wait after a flow is closed (usually zero).
+# amount of time to wait after a flow is closed (usually zero). "bypassed" is
```

Comment:
"bypass timeout controls locally bypassed flows. For these flows we don't do any other tracking. If no packets have been seen after this timeout, the flow is discarded".

Something like this.